.travis.yml

```diff
@@ -53,10 +55,10 @@ before_install:
     # it is a Pull Request related to Kafka (discovered from the PR title)
     # or if the build is from the "master" branch
     minikube_kafka)
-      if [[ "$TRAVIS_PULL_REQUEST" != false ]]; then
-        pr_kafka_title=$(curl -H "Authorization: token ${GITHUB_TOKEN}" "https://api.github.com/repos/$TRAVIS_REPO_SLUG/pulls/${TRAVIS_PULL_REQUEST}" | grep title | grep -i kafka || true)
-      fi
-      if [[ "$TRAVIS_PULL_REQUEST" == false || "$pr_kafka_title" != "" ]]; then
+      # if [[ "$TRAVIS_PULL_REQUEST" != false ]]; then
+      #  pr_kafka_title=$(curl -H "Authorization: token ${GITHUB_TOKEN}" "https://api.github.com/repos/$TRAVIS_REPO_SLUG/pulls/${TRAVIS_PULL_REQUEST}" | grep title | grep -i kafka || true)
+      # fi
+      if [[ "$TRAVIS_PULL_REQUEST" != false || "$pr_kafka_title" != "" ]]; then
```

Comment:
we can remove the GITHUB_TOKEN to allow to execute this from external PRs:
```
if [[ "$TRAVIS_PULL_REQUEST" != false ]]; then
        pr_title=$(curl "https://api.github.com/repos/$TRAVIS_REPO_SLUG/pulls/${TRAVIS_PULL_REQUEST}" | grep title || true)
fi
if [[ "$TRAVIS_PULL_REQUEST" == false || "$pr_title" == "" || "$pr_title" =~ ^.*(Kafka|kafka|KAFKA).*$ ]]; then 
```
that way we execute the kafka test if it is not a PR or if we cannot retrieve the PR title or if the title contains "kafka"

Reply:
👍